spec/index.bs

```diff
@@ -3568,7 +3568,13 @@ The {{GPUMapMode}} flags determine how a {{GPUBuffer}} is mapped when calling
             <div data-timeline=device>
                 [=Device timeline=] steps:
 
-                1. If |this| is [$invalid$], return.
+                1. If any of the following conditions are unsatisfied, return.
+
+                    <div class=validusage>
+                        - |this| is [$valid to use with$] |this|.{{GPUObjectBase/[[device]]}}.
+                        - |this|.{{GPUBuffer/[[internal state]]}} is "[=GPUBuffer/[[internal state]]/unavailable=]".
+                    </div>
+
```

Comment:
If this is guaranteed by previous steps, and I think it is, then we should write an `[=Assert=]` style assertion instead.

```suggestion
                    </div>

                1. [=Assert=] |this|.{{GPUBuffer/[[internal state]]}} is "[=GPUBuffer/[[internal state]]/unavailable=]".
```